server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -185,7 +185,14 @@ export class ScribeLambda implements IPartitionLambda {
                 }
 
                 this.clearCache = false;
-                if (value.operation.type === MessageType.Summarize) {
+
+                const isSummarizeMessage = value.operation.type === MessageType.Summarize;
+
+                // skip summarize messages that deli already acked
+                const shouldSkipSummarizeMessage = isSummarizeMessage && value.operation.serverMetadata &&
+                    typeof (value.operation.serverMetadata) === "object" && value.operation.serverMetadata.deliAcked;
```

Comment:
I think you can skip all the other type checks and just do value.operation?.serverMetadata?.deliAcked
Basically, everything is an object in JS.
But there are some funny things, like
```ts
const a = 5;
a.toString(); // works just fine
5?.toString(); // works just fine
5.toString(); // this will fail - compile and runtime - go figure.
```